vocabularies/identifier-type.ttl

```diff
@@ -0,0 +1,122 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix idt: <http://linked.data.gov.au/def/identifier-type/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/identifier-type> a owl:Ontology .
+
+idt:conceptScheme a skos:ConceptScheme ;
+ dct:created "2020-03-24T14:56:46+10:00"^^xsd:dateTime ;
+ dct:creator <http://linked.data.gov.au/org/gsq> ;
+ dct:modified "2020-03-24T14:57:04+10:00"^^xsd:dateTime ;
+ dct:source "Compiled by the Geological Survey of Queensland." ;
+ skos:definition "Types of Identification documents and numbers used in Queensland as acceptable proof of identity for individuals, businesses, and companies."@en ;
+ skos:prefLabel "Identifier Type"@en .
+
+<http://linked.data.gov.au/org/gsq>
+ a sdo:Organization ;
+ sdo:name "Geological Survey of Queensland" ; 
+ .
+
+idt:abn a skos:Concept ;
+ skos:altLabel "Australian business number"@en ;
+ skos:definition "The Australian Business Number (ABN) is a unique number used to identify business names and companies. The Australian Business Register (ABR) provides ABNs. Source:ASIC"@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "ABN"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:acn a skos:Concept ;
+ skos:altLabel "Australian company number"@en ;
+ skos:definition "Every company in Australia is issued with a unique, nine-digit number when registered. This is an Australian Company Number (ACN) and must be displayed on all company documents. Source:ASIC"@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "ACN"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:australian-visa a skos:Concept ;

```

Comment:
I don't think this is a party identifier

Reply:
valid QGOV document